README-iOS.md

```diff
@@ -0,0 +1,93 @@
+# VCMI for iOS
+
+You can run VCMI on iOS 10.0 and later, all devices are supported.
+
+## Installation guide
+
+Daily builds: https://builds.vcmi.download/branch/develop/iOS/
+
+To run on a non-jailbroken device you need to sign the IPA, you have the following options:
+
+- [AltStore](https://altstore.io/) or [Sideloadly](https://sideloadly.io/) - can be used on Windows too, don't require Xcode and dealing with signing on your own
+- get signer tool [here](http://dantheman827.github.io/ios-app-signer/) and a guide [here](https://forum.kodi.tv/showthread.php?tid=245978) (it's for Kodi, but the logic is the same). Signing with this app can only be done on macOS.
+- [create signing assets on macOS from terminal](https://github.com/kambala-decapitator/xcode-auto-signing-assets). In the command replace `your.bundle.id` with something like `com.MY-NAME.vcmi`.
+- [sign from any OS](https://github.com/indygreg/PyOxidizer/tree/main/apple-codesign). You'd still need to find a way to create signing assets (private key and provisioning profile) though.
+
+To play the game, you need to upload HoMM3 data files ("Data", "Maps" and "Mp3" directories) to the device. Use Finder / iTunes / Xcode for that. You can also add various [mods](https://wiki.vcmi.eu/Mod_list) by uploading Mods directory.
```

Comment:
Better to describe this moment in details, like step-by-step how do to this

Reply:
will find some manual on the internet and link to it

Reply:
done